tests/v2/validation/shell/shell_test.go

```diff
@@ -43,9 +43,7 @@ func (s *ShellTestSuite) SetupSuite() {
 	require.NoError(s.T(), err)
 
 	s.client = client
-
-	// Get clusterName from config yaml
-	s.clusterName = client.RancherConfig.ClusterName
+	s.clusterName = "local"
```

Comment:
Instead of this, can you do this?

- Remove clusterName from the struct
- Add a constant for the local cluster
- Use the constant in place of `s.clusterName` in the code